app/helpers/application_helper.rb

```diff
@@ -72,4 +72,28 @@ def content_tag_if(condition, *args, &block)
   def app_version_number
     @app_version ||= File.read(File.join(Rails.root, "VERSION"))
   end
+
+  def division_select_options(include_root: true, include_all: false, public: false)
```

Comment:
Can we default to the more restrictive option (`public_only: true` [and also change `public` to `public_only`]) and change the other calls to match? 

In general it's a best practice, when you're dealing with collections that can either be more or less restricted, to make the default the more restricted option, that way there's less of a chance that you'll leak any extra information.

Reply:
thank you